### ISSUE TITLE
pledging to protect city state quest giving 403 gp points for each great person (balance issues)

### ISSUE THREAD

**Issue by kawyua** [LoneGazebo/Community-Patch-DLL#4520]
sorry for editing after posting
_1. Mod version (i.e Date - 4/23):_
6/14

_2. Mod list (if using Vox Populi only, leave blank):_
all the mods of Vox Populi with eui
info addict

_3. Error description:_ 
403* gp points for each type of great person. city state quest. Some of these quests just need to be scaled down. gp points of each needs to have much lower ranges. 
Playing standard map size, quick speed, playing as the huns, communitas. 

_4. Steps to reproduce (optional):_
just the pledge to protect city state quest with foreign service

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):
[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/2161043/Logs.zip)

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):
[AutoSave_0103 AD-0860.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/2161037/AutoSave_0103.AD-0860.zip)
[AutoSave_0102 AD-0840.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/2161038/AutoSave_0102.AD-0840.zip)

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):
![citystateprotection2](https://user-images.githubusercontent.com/12496046/42244979-1f647900-7ee5-11e8-8eaf-c0b8a2cd89b4.PNG)
![citystateprotection1](https://user-images.githubusercontent.com/12496046/42244980-1f772b22-7ee5-11e8-8761-8942e80ec0b0.PNG)
![citystateprotection3](https://user-images.githubusercontent.com/12496046/42246603-33378f70-7eeb-11e8-882b-ad0646d630c3.PNG)

**Comment by TechpriestEnginseer** [comment]
Pledging to Protect is a difficult task to do. Needing a strong military and "reasons to pledge" in the first place. Not to mention they don't scale the more GP you have, so they only help civs who expand through military.

**Comment by kawyua** [comment]
sorry, my post was first a confusion where I had no clue how I got 6 great persons out of nowhere. I figured out what happened and then edited my post because I thought it was out of line. =( 

I've seen these rewards before and a city state reward that is ~2400 gp points is pretty big for a quick speed game. ~400 was something like the upper bound for the time period I was playing (this is with the statecraft +50% city state rewards policy) and I did have some good luck. Still, that 400 is multiplied by 6 and that is a pretty big difference. great person points are pretty damn nearly 1:1 as valuable as any other yield. 

I'll close this because I guess people disagree.
